WMIAdventure/backend/WMIAdventure_backend/IngameUsers/businesslogic/experience/tests/test_Experience.py

```diff
@@ -0,0 +1,32 @@
+import unittest
+
+from IngameUsers.businesslogic.experience.Experience import Experience
+from IngameUsers.businesslogic.experience.formulas import level_to_exp
+
+
+class ExperienceTestCase(unittest.TestCase):
+    def test_should_transform_exp_to_level_correctly(self):
+        expected_level = 20
+        exp = level_to_exp(expected_level) + 1
+        experience_obj = Experience(exp)
+        self.assertEqual(experience_obj.level, expected_level)
+
+    def test_should_transform_exp_to_level_correctly_first_level(self):
+        expected_level = 1
+        exp = 1
```

Comment:
```suggestion
        exp = 0
```
to się tyczy tej mojej sugestii z wcześniej

Reply:
Tutaj to nie ma znaczenia, czy mamy 0 expa czy 1, oba te casy mają level 1

Reply:
racja